content/docs/sidebar.json

```diff
@@ -94,13 +94,18 @@
     "slug": "user-guide",
     "source": "user-guide/index.md",
     "children": [
+      {
+        "label": "Glossary",
+        "slug": "glossary",
+        "source": "basic-concepts"
+      },
       {
         "label": "What is DVC?",
         "slug": "what-is-dvc"
       },
       {
         "slug": "project-structure",
-        "source": "user-guide/project-structure/index.md",
+        "source": "project-structure/index.md",
```

Comment:
I noticed that this page and the page on line `143` had incorrect `source` causing the "Edit on Github" link to take you to a 404 page. I went ahead and fixed them.

Reply:
Interesting. Should the engine throw an error when building? Out of scope for this PR but could open an issue (or move the fix to another PR and we can decide there).

Reply:
I'm not sure about the engine throwing an error, but I don't mind moving these fixes to a separate PR if needed!